### ISSUE TITLE
Improve kpkt() choice of frequency for bound-free emission 

### ISSUE THREAD

**Issue by jhmatthews** [sirocco-rt/sirocco#401]
Currently, the routine kpkt() assumes a hydrogenic cross-section when deciding on the frequency for the photon that is produced via recombination. This should be improved to instead take into account the actual shape of the cross-section in question, which in principle should be a matter of using existing machinery in the code. 

This occurs around line 984 of matom.c

```C
        /* Now (as in matom) choose a frequency for the new packet. */

        p->freq = phot_top[i].freq[0] - (log (1. - random_number(0.0,1.0)) * xplasma->t_e / H_OVER_K);
		
        /* Co-moving frequency - changed to rest frequency by doppler */
        /*Currently this assumed hydrogenic shape cross-section - Improve */
```

**Comment by jhmatthews** [comment]
This has now been done in the bf_simple_alternate branch but will not be merged until we have understood the issue with our "reweighting" for bound-free simple continua (see #394 and evernote docs).

**Comment by jhmatthews** [comment]
In principle this is fixed by #433 but wait for testing before we close this issue.

**Comment by jhmatthews** [comment]
This is being closed since it's now implemented, but with the caveat that simple ion recombination continua should really be tested more thoroughly in macro-atom line transfer mode (or "indivisible mode", more correctly). However, they should be dealt with more accurately than before. 

To test this, one needs to look at a non-hydrogenic, simple-ion and compare it's continuum to ground truth. This could be done, for example, by looking at an He I recombination continuum and comparing standard mode, a run with a He macro-atom and indivisible He simple ion. Alternatively, one can compare to an analytic expression.
